scrapers/nserc/scrape_details.sh

```
@@ -0,0 +1,69 @@
+    echo "File $output_file already exists. Skipping..."
+    return 0
+  fi
+  sleep 0.25
+
+  echo "Downloading details for ID: $id"
+  curl "https://www.nserc-crsng.gc.ca/ase-oro/Details-Detailles_eng.asp?id=${id}" \
+    --compressed \
+    -s \
+    --connect-timeout 3 \
+    --max-time 3
+    -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:136.0) Gecko/20100101 Firefox/136.0' \
```

Shellcheck (0.10.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)